common/api/summary/appui-react.exports.csv

```diff
@@ -186,7 +196,6 @@ public;CursorPopup
 public;CursorPopupContent(props: CommonDivProps): React_2.JSX.Element
 internal;CursorPopupFadeOutEventArgs
 public;CursorPopupManager
-alpha;CursorPopupMenu 
```

Comment:
Wow, I am seeing this for the first time and have no idea. I would have assumed that a module was removed from the barrel file unintentionally, but that's not the case. Since it is at least in the API markdown file, I think we can ignore it as a tooling issue for now?